### ISSUE TITLE
Parallelism and HPC Julia page is not scrollable. 

### ISSUE THREAD

**Issue by mmahrouss** [UCIDataScienceInitiative/IntroToJulia#46]
When the following link opens the browser can't scroll through the page
http://ucidatascienceinitiative.github.io/IntroToJulia/Slides/HPCJulia#/ 

Tried on Google Chrome and Firefox on Ubuntu 18.04

**Comment by ChrisRackauckas** [comment]
Yeah, the slides don't scroll, but the HTML pages do. I'm not sure this issue can be fixed with Jupyter's output.
